modules/beam/src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/AssetsManagement.scala

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2012-2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.beam
+
+import java.io.File
+import java.net.URI
+import java.lang.{Long => JLong}
+
+import cats.syntax.either._
+
+import org.joda.time.Duration
+
+import org.slf4j.LoggerFactory
+
+import org.apache.beam.sdk.extensions.gcp.options.GcsOptions
+import org.apache.beam.sdk.io.GenerateSequence
+import org.apache.beam.sdk.values.WindowingStrategy.AccumulationMode
+import org.apache.beam.sdk.transforms.windowing.{AfterPane, Repeatedly}
+import org.apache.beam.sdk.transforms.windowing.Window.ClosingBehavior
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.coders.Coder
+import com.spotify.scio.util.RemoteFileUtil
+import com.spotify.scio.values.{DistCache, SCollection, SideInput, WindowOptions}
+
+import com.snowplowanalytics.snowplow.enrich.common.enrichments.registry.EnrichmentConf
+import com.snowplowanalytics.snowplow.enrich.beam.utils.createSymLink
+
+/**
+ * Module responsible for assets (such as MaxMind and referer-parser DBs)
+ * management: downloading, distributing per workers, updating when necessary
+ */
+object AssetsManagement {
+
+  val SideInputName = "assets-refresh-tick"
```

Comment:
Yes! At least tests need to refer to it: https://github.com/snowplow/enrich/pull/322/files/2c28aeb995324e5d5363dd3a897105d8cb73b964#diff-634f9ba9e2b3f5c184fae17dbd4216c3R155, otherwise fail with runtime exceptioin